nodejs/listeners/actions/block_app-home-nav-completed.js

```diff
@@ -1,13 +1,13 @@
 const reloadAppHome = require('../../utilities/reload-app-home');
 
-module.exports = (app) => {
-  app.action({ action_id: 'app-home-nav-completed', type: 'block_actions' }, async ({ body, ack, client }) => {
-    try {
-      await ack();
-      await reloadAppHome(client, body.user.id, body.team.id, 'completed');
-    } catch (error) {
-      // eslint-disable-next-line no-console
-      console.error(error);
-    }
-  });
+const appHomeNavCompletedCallback = async ({ body, ack, client }) => {
```

Comment:
Interesting approach to split out the callbacks from the listener. This does lead to more testable code. We have never shown this as a best practice but maybe we should?